Content.Server/Light/EntitySystems/EmergencyLightSystem.cs

```diff
@@ -172,15 +172,26 @@ public void UpdateState(EntityUid uid, EmergencyLightComponent component)
         if (!TryComp<ApcPowerReceiverComponent>(uid, out var receiver))
             return;
 
-        if (receiver.Powered && !component.ForciblyEnabled)
+        if (!TryComp<AlertLevelComponent>(_station.GetOwningStation(uid), out var alerts))
+            return;
+
+        if (alerts.AlertLevels == null || !alerts.AlertLevels.Levels.TryGetValue(alerts.CurrentLevel, out var details))
+            return;
+
+        if (receiver.Powered && !component.ForciblyEnabled) // Green alert/empty battery 
```

Comment:
Emergency lights shouldn't just fail because of no alerts.

Reply:
if alerts are null, the lights will default to off and red.